sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CopyAuthorization.java

```diff
@@ -35,17 +37,20 @@
     /*
      * Resource Identifier.
      */
+    @JsonIgnore
```

Comment:
we should have this 2, otherwise the BeginCopyMethod will fail with missing properties

Reply:
We still return these values in the object creation. They are just ignored when we jsonify the object.

Reply:
We still need them when we create the json string. If I ask you for an uth token so I cam copy my model into your account and you give me a string with only modelid, access token, and expiration date, my StartCopyModel in .NET will fail.

Reply:
See the string I mentioned before:
```
{"modelId": "26f7c09b-3e1d-4348-b98d-18187afcbf17", "accessToken": "82ea8d63-1eaa-498c-bd8b-xxxxxxxx", "expirationDateTimeTicks": 637251564427472980, "resourceId": "xxxxx", "resourceRegion": "westus2"}
```